### PR TITLE
Migrate inspire schemas

### PR DESCRIPTION
**Pull request by JohannaOtt**
SVC-1946

### CODE REVIEW

File: annex-2-3/mappings/LandCover/ALKIS2INSPIRE_LC_v2.halex.alignment.xml [hunk 1]
```diff
@@ -982,6 +982,16 @@ return c.extent.value()
...
     </cell>
+    <cell relation="eu.esdihumboldt.hale.align.assign" id="C4c1a3776-4a5f-4fff-8e66-3b3675020ed7" priority="normal">
+        <target>
+            <property>
+                <type name="LandCoverUnitType" ns="http://inspire.ec.europa.eu/schemas/lcv/5.0"/>
+                <child name="dataset" ns="http://inspire.ec.europa.eu/schemas/lcv/5.0"/>
+                <child name="href" ns="http://www.w3.org/1999/xlink"/>
+            </property>
+        </target>
+        <parameter value="#{{project:LANDCOVER_DATASET}}" name="value"/>
```
Comment (annat2022) [inline annex-2-3/mappings/LandCover/ALKIS2INSPIRE_LC_v2.halex.alignment.xml:993]: As far as I can see this change does not fit to the commit message? It's not about the import of codelists?

File: annex-2-3/mappings/LandCover/v1.0.1/LB-Landbedeckung1.0.1_INSPIRE-LC.halex.alignment.xml [hunk 2]
```diff
@@ -721,6 +721,16 @@ return c.extent.value()
...
     </cell>
+    <cell relation="eu.esdihumboldt.hale.align.assign" id="C81cf9772-5988-405b-b53a-9ef95598ad1f" priority="normal">
+        <target>
+            <property>
+                <type name="LandCoverUnitType" ns="http://inspire.ec.europa.eu/schemas/lcv/5.0"/>
+                <child name="dataset" ns="http://inspire.ec.europa.eu/schemas/lcv/5.0"/>
+                <child name="href" ns="http://www.w3.org/1999/xlink"/>
+            </property>
+        </target>
+        <parameter value="#{{project:LANDCOVER_DATASET}}" name="value"/>
```
Comment (annat2022) [inline annex-2-3/mappings/LandCover/v1.0.1/LB-Landbedeckung1.0.1_INSPIRE-LC.halex.alignment.xml:732]: Same here, the change doesn't fit to the commit message?

**Review by annat2022 — COMMENTED** [review]
LGTM, expect the the comments I added directly (but that's only about the changes not fitting to the commit message.)

File: annex-2-3/mappings/LandCover/ALKIS2INSPIRE_LC_v2.halex.alignment.xml (hunk 1 above)
Comment (JohannaOtt) [reply annex-2-3/mappings/LandCover/ALKIS2INSPIRE_LC_v2.halex.alignment.xml:993]: Good catch, removed it from the codelist related commit now and added an own commit for these changes.

File: annex-2-3/mappings/LandCover/v1.0.1/LB-Landbedeckung1.0.1_INSPIRE-LC.halex.alignment.xml (hunk 2 above)
Comment (JohannaOtt) [reply annex-2-3/mappings/LandCover/v1.0.1/LB-Landbedeckung1.0.1_INSPIRE-LC.halex.alignment.xml:732]: Good catch, removed it from the codelist related commit now and added an own commit for these changes.

**Review by annat2022 — APPROVED** [review]
LGTM

**Pull request merged**